src/api/api.js

```diff
@@ -1,30 +1,58 @@
 // import axios from 'axios';
-import { events } from './data';
-
-// const BASE_URL = 'https://random_url.com';
-// const OK = 200;
+import { people } from './data';
+import { getEvents } from '../back_end/api/end_points.js';
 
 // For dummy endpoints
 const delay = ms => new Promise(
     resolve => setTimeout(resolve, ms)
   );
 
+const isValidHttpUrl = (string) => {

```

Comment:
I see you copy pasted :D 
Y'all really don't add commenting in your code hm

Reply:
Stack overflow= God tier code

Reply:
Tru